src/test/java/uk/gov/digital/ho/hocs/info/api/UserIntegrationTest.java

```diff
@@ -48,41 +41,43 @@ public class UserIntegrationTest {
     @Autowired
     ObjectMapper mapper;
 
-    @Before
+    @BeforeEach
     public void setup() throws IOException {
         headers = new HttpHeaders();
         headers.add(HttpHeaders.ACCEPT, ContentType.APPLICATION_JSON.toString());
     }
 
     @Test
     public void shouldCreateUser() {
-
         //create
-        CreateUserDto createUserDto = new CreateUserDto("aa" + System.currentTimeMillis() + "@homeoffice.gov.uk", "Bbbbb", "Ccccc");
+        var createUserDto = new CreateUserDto("1@example.com", "A", "B");
```

Comment:
using var here but not elsewhere?

Reply:
I'm consistently inconsistent, I forget it exists and then I don't sometimes. I'll fix up to be consistent.